staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -111,17 +115,21 @@ func (t *azureDiskCSITranslator) TranslateInTreePVToCSI(pv *v1.PersistentVolume)
 		VolumeHandle:     azureSource.DataDiskURI,
 		ReadOnly:         *azureSource.ReadOnly,
 		FSType:           *azureSource.FSType,
-		VolumeAttributes: map[string]string{},
+		VolumeAttributes: map[string]string{azureDiskKind: "Managed"},
 	}
 
-	if *azureSource.CachingMode != "" {
+	if azureSource.CachingMode != nil {
 		csiSource.VolumeAttributes[azureDiskCachingMode] = string(*azureSource.CachingMode)
 	}
 
-	if *azureSource.FSType != "" {
+	if azureSource.FSType != nil {
```

Comment:
also check for empty string here? Is there any default value if it is empty string?

Reply:
Oh, I see it's already checked below.